All In/Features/Profile/ProfileView.swift

```diff
@@ -13,10 +13,13 @@ struct ProfileView: View {
     @Binding var tabSelection: Int
 
     @State private var user: User
+    @State private var editingUsername = false
```

Comment:
nit: Usually the convention is to use "is" or "did"  in front to indicate that it is a boolean value so "isEditingUsername" or "didEditUsername"